Cargo.toml

```diff
@@ -19,7 +19,7 @@ geo-types = { version = "0.7.10", optional = true }
 libc = "0.2.119"
 num-traits = "0.2.14"
 thiserror = "1.0.30"
-reqwest = { version = "0.12.0", optional = true, default-features = false, features = ["blocking", "rustls-tls"] }
+ureq = { version = "2.0.0", optional = true }
```

Comment:
Did you actually test this on 2.0.0? Otherwise please specify whatever version you tested on (latest is 2.9.7).

Otherwise it's hard to know if you are leveraging any functionality that was added in a minor release.

Reply:
Yes, I tested it on 2.0.0.